platforms/anchor-platform/admin-guide/sep24/faq.mdx

```diff
@@ -3,9 +3,9 @@ title: "FAQ"
 sidebar_position: 50
 ---
 
-### How To Use Interactive JWTs?
+### How To Use JWTs?
```

Comment:
nit: Can we change this to `Using JWTs`?

Reply:
The JWT is for interactive URL though. Maybe `How to use JWTs of interactive URL`?

Reply:
Isn't it the same for more_info_url jwt as well? If no what's the difference?

Reply:
Yeah, it's the same for `more_info_url`. You're right it's in the FAQ section so phrasing it like a question makes sense. I'm fine with leaving it as is.